moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1925,9 +1925,6 @@ double moveit::core::RobotState::computeCartesianPath(const JointModelGroup* gro
   traj.clear();
```

Comment:
In the single waypoint case the additional steps are required for the testing to work. In the multi waypoint case test_joint_space_jump is false because jump_threshold is 0.0, so only one step is required per waypoint.